submissions/kt-std/js-dom/js/script.js

```diff
@@ -0,0 +1,113 @@
+const ASSETS_PATH = "assets/";
```

Comment:
Do you really still need this file?

Reply:
The ASSETS_PATH variable is needed to set background images depending on the screen width. It sets the part of the path to where the images are stored. I guess there is no need to remove it because, in that case, it won't work / the path will have to be set explicitly, which is not a good idea, am I right?